env_test.go

```diff
@@ -364,6 +364,31 @@ func TestErrorRequiredNotSet(t *testing.T) {
 	assert.Error(t, env.Parse(cfg))
 }
 
+func TestParseExpandOption(t *testing.T) {
+	type config struct {
+		Host        string `env:"HOST" envDefault:"localhost"`
+		Port        int    `env:"PORT,expand" envDefault:"3000"`
+		SecretKey   string `env:"SECRET_KEY,expand"`
+		ExpandKey   string `env:"EXPAND_KEY"`
+		CompoundKey string `env:"HOST_PORT,expand"`
+	}
+	os.Setenv("HOST", "localhost")
```

Comment:
I was hoping to rely on `envDefault` for `HOST` and `PORT` but the package does not call `os.Setenv` when parsing the config.